features/guestfs/config.mk

```diff
@@ -16,7 +16,7 @@ GUESTFS_FILE_DATABASE = \
 
 GUESTFS_PROGS = cpio blockdev btrfs-image btrfsck btrfstune \
 	mklost+found mkswap tune2fs base64 chattr cksum lsattr \
-	md5sum sha1sum sha224sum sha256sum sha384sum sha512sum \
+	md5sum sha1sum sha224sum sha256sum sha384sum sha512sum gostsum gost12sum \
```

Comment:
Can we do something like this to avoid strict dependence on these utilities?

```diff
diff --git a/features/guestfs/config.mk b/features/guestfs/config.mk
index 2eaf4015..2f04c3a8 100644
--- a/features/guestfs/config.mk
+++ b/features/guestfs/config.mk
@@ -36,7 +36,9 @@ GUESTFS_PROGS_PATTERNS = \
        */*label \
        */*resize* \
        *disk \
-       */hivex*
+       */hivex* \
+       */gostsum \
+       */gost12sum
 
 GUESTFS_FILES = \
        $(GUESTFS_FILE_DATABASE) \
```